woocommerce/woocommerce-ios
# System Status Report: Format the report to match with Core's

Part of #5701


### Description

This PR takes care of formatting of the fetched system status to a friendly format.  It also includes updates of the models in Networking for public access control and fixes some parsing issues. 

### Testing instructions

- Log in to a valid account
- On My Store tab, navigate to Settings.
- Select Help & Support > System Status Report.
- Notice that the report is fetched successfully and formatted to the matching style with Core's.

### Screenshots

<img src="https://user-images.githubusercontent.com/5533851/146485263-acf3434a-5d16-471b-bf38-ceb5c93aefd9.png" width=320 />

---
